### PR TITLE
[SP-5461] 

### PR DESCRIPTION
**Pull request by InbarPerry**

### CODE REVIEW

**Review by oriser — APPROVED** [review]
LGTM

File: steps/http/request/input.go [hunk 1]
```diff
@@ -36,12 +38,18 @@ const (
...
 
-	defaultScheme = "https"
+	defaultScheme            = "https"
+	defaultTimeout           = 30
+	defaultContentType       = "application/json; charset=utf-8"
+	defaultAuthorization     = string(None)
+	defaultResponseAPIObject = true
+
+	maxDuration = 3 * time.Minute
+	minDuration = 30 * time.Second
```
Comment (hagaishapira) [inline steps/http/request/input.go:48]: Why is this the minimum?

File: steps/http/request/input.go [hunk 2]
```diff
@@ -36,12 +38,18 @@ const (
...
 	ResponseAPIObjectOnly = "RESPONSE_API_OBJECT_ONLY"
 
-	defaultScheme = "https"
+	defaultScheme            = "https"
+	defaultTimeout           = 30
+	defaultContentType       = "application/json; charset=utf-8"
+	defaultAuthorization     = string(None)
+	defaultResponseAPIObject = true
+
+	maxDuration = 3 * time.Minute
```
Comment (hagaishapira) [inline steps/http/request/input.go:47]: and this the max?

**Review by hagaishapira — APPROVED** [review]
LGTM

**Pull request merged**